content/docs/studio/user-guide/projects-and-experiments/live-metrics.md

```diff
@@ -0,0 +1,107 @@
+# Generate live (real-time) metrics for experiments
+
+When you
+[submit a new experiment](/doc/studio/user-guide/projects-and-experiments/run-experiments)
+from Iterative Studio, the CI/CD setup in your Git repository gets invoked. If
+this setup includes a model training process, it will be triggered.
+
+In this model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This requires a 3-step process:
```

Comment:
Why do we want to send updates to Studio bypassing the Git repo. Shouldn't Git be the source of truth? Where will logs be stored, DVC remote? Thanks

Reply:
Addressed in https://github.com/iterative/dvc.org/pull/4012/commits/22e52337f839ef931e18531b66b56eefe4b5ebfb

Reply:
> This will help to keep your Git commit history unpolluted, as you
can push only the final results of your experiments while being able to view all
intermediate results in Studio (in the form of live metrics and plots).

Still unclear in general where logs will be stored. Doesn't DVCLive use a directory in the repo normally?

Reply:
You can choose not to commit / push the logs to your repo. So my understanding is that when you are running experiments, you will push the live metrics and plots to Studio but won't commit / push the DVCLive output. @daavoo is this explanation correct?

Reply:
Considering the MVP, I don't think there is a reason to say `This will help to keep your Git commit history unpolluted, as you can push only the final results of your experiments`. 

The MVP doesn't really help with keeping Git commit history clean and I would drop those referneces (at least until we complete the follow-up proposal about local experiments)

The MVP value is `being able to view all intermediate results in Studio` (I would add: `while the experiment is running).

Reply:
Updated.